inbm-lib/inbm_common_lib/utility.py

```diff
@@ -124,7 +124,7 @@ def remove_file(path: Union[str, Path]) -> bool:
         return False
 
 
-def create_file_with_contents(path: Union[str, Path], contents: List[str]) -> None:
+def create_file_with_contents(path: Union[str, Path], contents: list[str]) -> None:
```

Comment:
Ensure that the `create_file_with_contents` function handles the case where `contents` is an empty list to avoid creating empty files unintentionally. [medium]